src/System.Net.HttpListener/tests/SimpleHttpTests.cs

```diff
@@ -75,29 +79,33 @@ public void BasicTest_StartAbortClose_NoException()
             }
         }
 
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
+        [Fact]
+        [ActiveIssue(20009, TargetFrameworkMonikers.Uap)]
         public void BasicTest_StopNoStart_NoException()
         {
             HttpListener listener = new HttpListener();
             listener.Stop();
             listener.Close();
         }
 
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
+        [Fact]
+        [ActiveIssue(20009, TargetFrameworkMonikers.Uap)]
```

Comment:
It's not relevant: we just need to track them against the Networking UWP epics / tracking issues hierarchy to ensure we're getting everything done :).

Thanks for changing @joperezr !